cmd/ratify/cmd/referrer.go

```diff
@@ -198,10 +202,27 @@ func showRefManifest(opts referrerCmdOptions) error {
 		return err
 	}
 
-	desc := ocispecs.ReferenceDescriptor{Descriptor: v1.Descriptor{Digest: digest}}
+	ref := fmt.Sprintf("%s@%s", subRef.Path, digest)
+
+	manifestRef, err := utils.ParseSubjectReference(ref)
+	if err != nil {
+		return err
+	}
+
+	ctx := context.Background()
 	for _, referrerStore := range stores {
 		if referrerStore.Name() == opts.storeName {
-			manifest, err := referrerStore.GetReferenceManifest(context.Background(), subRef, desc)
+			manifestDesc, err := referrerStore.GetSubjectDescriptor(ctx, manifestRef)
```

Comment:
Wouldn't the artifact type be returned as part of the reference manifest descriptor?